modules/self_managed_nodes/variables.tf

```diff
@@ -88,3 +82,9 @@ variable "force_irsa" {
   type        = bool
   default     = true
 }
+
+variable "tags" {
+  description = "Tags for all resources"
+  type        = map(string)
+  default     = {}
+}
```

Comment:
Add newline at the end of file.

Reply:
@niroz89 I ran terraform fmt command but anyway still added the space as you suggested.